gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oauth2/service/granter/CompositeTokenGranter.java

```diff
@@ -103,8 +103,6 @@ public class CompositeTokenGranter implements TokenGranter, InitializingBean {
     @Autowired
     private AuditService auditService;
 
-    public CompositeTokenGranter() { }
```

Comment:
Are you sure it is not needed to initialize bean?

Reply:
If there's no other constructor, java creates a no-arg constrcutor implicitly, so there's no need to declare it.